library/deployment.go

```diff
@@ -13,16 +13,18 @@ import (
 // swagger:model Deployment
 type Deployment struct {
 	ID          *int64              `json:"id,omitempty"`
+	Number      *int64              `json:"number,omitempty"`
 	RepoID      *int64              `json:"repo_id,omitempty"`
 	URL         *string             `json:"url,omitempty"`
-	User        *string             `json:"user,omitempty"`
```

Comment:
just curious, why'd we remove User?

Reply:
because I added a CreatedBy field instead